elastic/kibana
# [Infra] [APM] Use bottom bar action buttons in infra settings

Closes #169210 
## Summary

This PR adds bottom bar action buttons to the infra settings page to make it consistent with the APM settings page. The bottom bar appears after a change is made (and won't appear if the value is the same as it was before - same as APM).

| APM | Infra Now |
| ---- | ---------- |
|  ![image](https://github.com/elastic/kibana/assets/14139027/8400cc75-c3d3-40c6-adf7-040779598c7d) | ![image](https://github.com/elastic/kibana/assets/14139027/60f49a78-f733-43f4-a1c8-9b9c337b49f6) |

The infra page validation is a bit different so I used the same logic as in advance settings - if the field is invalid disable the button and show a tooltip: 

<img width="1920" alt="image" src="https://github.com/elastic/kibana/assets/14139027/9e73e77f-f572-4dbe-a8cc-9aa7bf5ec370">

## Testing 

- Open the infra settings page
   - Check if the bottom bar is displayed and the save/discard functionality works as expected
   - Check infra valdation
- Verify that the APM settings page works as expected


https://github.com/elastic/kibana/assets/14139027/6804e24b-2b33-456d-be8f-cb17cc044759
